asmcli/components/ca/managed-cas.sh

```diff
@@ -88,9 +99,9 @@ x_wait_for_enabling_workload_certificates() {
   # Enabling workload certificate feature on a cluster will result in
```

Comment:
1500 second is 25 minutes

Reply:
Reverted to 900 based on the comment above.